recipes-bsp/u-boot/files/ae350-ax45mp/0003-riscv-ae350-Fix-XIP-config-boot-failure.patch

```diff
@@ -0,0 +1,79 @@
+From f5b63c98181df2d0f667439f36b9416671d41fc8 Mon Sep 17 00:00:00 2001
+From: Leo Yu-Chi Liang <ycliang@andestech.com>
+Date: Wed, 1 Jun 2022 10:01:49 +0800
+Subject: [PATCH] riscv: ae350: Fix XIP config boot failure
+
+The booting flow is SPL -> OpenSBI -> U-Boot.
+The boot hart may change after OpenSBI and may not always be hart0,
+so wrap the related branch instruction with M-MODE.
+
+Current DTB setup for XIP is not valid.
+There is no chance for CONFIG_SYS_FDT_BASE, the DTB address used
+in XIP mode, to be returned. Fix this.
+
+Fixes: 2e8d2f88439d ("riscv: Remove OF_PRIOR_STAGE from RISC-V boards")
+Signed-off-by: Rick Chen <rick@andestech.com>
+Signed-off-by: Leo Yu-Chi Liang <ycliang@andestech.com>
+
+The patch was imported from patchwork:
+https://patchwork.ozlabs.org/project/uboot/patch/20220601020149.11805-1-ycliang@andestech.com/
+
+Signed-off-by: Yu Chien Peter Lin <peterlin@andestech.com>
```

Comment:
Same here

Reply:
Hi @alistair23 

Can I just leave the patchwork link? If not, I have to wait for maintainer send the PR to U-boot master branch.

Reply:
It's still just submitted I think

Reply:
Ok.. I'll wait for acceptance.

Reply:
It doesn't have to be accepted. It can just be submitted instead https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Importing_from_Elsewhere

Reply:
The patches has been merged recently. The commit id added.